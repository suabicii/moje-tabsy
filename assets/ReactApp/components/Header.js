import React from "react";
import Sidebar from "./Sidebar";
import NavbarMain from "./NavbarMain";

function Header() {
    return (
        //Main Navigation-->
        <header className="mb-5">
            <Sidebar/>
            <NavbarMain/>
        </header>
        //Main Navigation-->
    );
}

export default Header;