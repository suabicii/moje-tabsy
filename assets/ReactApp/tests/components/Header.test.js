import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";
import Header from "../../components/Header";

it('should correctly render Header', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Header/>);

    expect(renderer.getRenderOutput()).toMatchSnapshot();
});