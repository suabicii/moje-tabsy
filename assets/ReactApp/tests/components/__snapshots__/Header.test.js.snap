// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Header 1`] = `
<div
  className="mb-5"
>
  <Sidebar />
  <NavbarMain />
</div>
`;
